Packs/CortexAttackSurfaceManagement/README.md

```diff
@@ -133,11 +133,11 @@ A playbook that given the IP address enriches Azure information relevant to ASM
 
 ![Cortex ASM - Azure Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Azure_Enrichment.png)
 
-#### Cortex ASM - Decision
+#### Cortex ASM - Cortex Endpoint Enrichment
 
-A playbook that returns "RemediationAction" options based on meeting "Automated Remediation Requirements" as well as whether ServiceNowV2 integration is set up.
+This playbook is used to pull information from Cortex Endpoint (XSIAM/XDR) systems for enrichment purposes.
 
-![Cortex ASM - Decision](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Decision.png)
+! TODO ADD LINK AFTER PR
```

Comment:
Don't forget this!

Reply:
@ShirleyDenkberg , you caught be slipping! LOL